### PR TITLE
feat: [LWG-1044]: Revamped designs for static schedules

### PR DESCRIPTION
**Pull request by rishik30**
##### Screenshots:

Screenshot 2021-12-21 at 10 32 34 AM (1)

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

File: src/modules/75-ce/common/FixedSchedule/FixedScheduleForm.tsx [hunk 1]
```diff
@@ -146,159 +151,45 @@ const FixedScheduleForm: React.FC<FixedScheduleFormProps> = props => {
...
-                        _formikProps.values.beginsOn
-                          ? `${getStaticSchedulePeriodTime(_formikProps.values.beginsOn)}`
-                          : undefined
-                      }
-                      onChange={(value, _err) => {
-                        _formikProps.setFieldValue('beginsOn', getStaticSchedulePeriodString(Number(value)))
+              <Container className={css.inputRow}>
+                <Layout.Horizontal spacing={'large'}>
+                  <Container style={{ flex: 1 }}>
+                    <Label>{'Type*'}</Label>
```
Comment (jenil-harness) [inline src/modules/75-ce/common/FixedSchedule/FixedScheduleForm.tsx:157]: To string constant.

File: src/modules/75-ce/common/FixedSchedule/FixedScheduleForm.tsx [hunk 2]
```diff
@@ -146,159 +151,45 @@ const FixedScheduleForm: React.FC<FixedScheduleFormProps> = props => {
...
-                      value={
-                        _formikProps.values.endsOn
-                          ? `${getStaticSchedulePeriodTime(_formikProps.values.endsOn)}`
-                          : undefined
-                      }
-                      onChange={(value, _err) => {
-                        _formikProps.setFieldValue('endsOn', getStaticSchedulePeriodString(Number(value)))
+                  </Container>
+                  <Container style={{ flex: 1 }}>
+                    <Label>{'Timezone*'}</Label>
```
Comment (jenil-harness) [inline src/modules/75-ce/common/FixedSchedule/FixedScheduleForm.tsx:181]: strings

File: src/modules/75-ce/common/FixedSchedule/FixedScheduleForm.tsx [hunk 3]
```diff
@@ -319,21 +210,191 @@ const FixedScheduleForm: React.FC<FixedScheduleFormProps> = props => {
...
+              }
+              onChange={handleEndDateChange}
+              data-testid="endsOn"
+            />
+          )}
+        </div>
+      </Layout.Horizontal>
+      <Checkbox
+        name={'neverEnds'}
+        label={'Never ends'}
```
Comment (jenil-harness) [inline src/modules/75-ce/common/FixedSchedule/FixedScheduleForm.tsx:290]: strings

File: src/modules/75-ce/common/FixedSchedule/FixedScheduleForm.tsx [hunk 4]
```diff
@@ -319,21 +210,191 @@ const FixedScheduleForm: React.FC<FixedScheduleFormProps> = props => {
...
+                  formikProps.setFieldValue('startTime', null)
+                  formikProps.setFieldValue('endTime', null)
+                  // formikProps.setFieldValue('allDay', false)
+                }
+              }}
+            />
+          </Layout.Horizontal>
+        </Container>
+        <Container style={{ flex: 1 }}>
+          <Label>{'Time'}</Label>
```
Comment (jenil-harness) [inline src/modules/75-ce/common/FixedSchedule/FixedScheduleForm.tsx:348]: Strings

File: src/modules/75-ce/common/FixedSchedule/FixedScheduleForm.tsx [hunk 5]
```diff
@@ -319,21 +210,191 @@ const FixedScheduleForm: React.FC<FixedScheduleFormProps> = props => {
...
+          <Layout.Horizontal flex={{ alignItems: 'baseline', justifyContent: 'start' }}>
+            <Radio
+              label=""
+              value={'sometime'}
+              checked={!formikProps.values.allDay}
+              onChange={_ => formikProps.setFieldValue('allDay', false)}
+            />
+            <Container>
+              <Layout.Horizontal flex={{ alignItems: 'center', justifyContent: 'start' }} className={css.fromTime}>
+                <Text>{'from'}</Text>
```
Comment (jenil-harness) [inline src/modules/75-ce/common/FixedSchedule/FixedScheduleForm.tsx:368]: strings

File: src/modules/75-ce/common/FixedSchedule/FixedScheduleForm.tsx [hunk 6]
```diff
@@ -319,21 +210,191 @@ const FixedScheduleForm: React.FC<FixedScheduleFormProps> = props => {
...
+                  useAmPm
+                  value={getTime(formikProps.values.startTime)}
+                  disabled={formikProps.values.allDay}
+                  onChange={d => {
+                    formikProps.setFieldValue('startTime', { hour: d.getHours(), min: d.getMinutes() })
+                  }}
+                />
+              </Layout.Horizontal>
+              <Layout.Horizontal flex={{ alignItems: 'center', justifyContent: 'start' }} className={css.toTime}>
+                <Text>{'to'}</Text>
```
Comment (jenil-harness) [inline src/modules/75-ce/common/FixedSchedule/FixedScheduleForm.tsx:379]: strings

**Review by jenil-harness — DISMISSED** [review]

**Comment by rishik30** [comment]
retrigger jest

**Comment by rishik30** [comment]
retrigger jest

**Review by shalinlk — APPROVED** [review]

**Review by jenil-harness — APPROVED** [review]

**Pull request merged**